std_msgs/Header header

string note

# confidence in note classification *not confidence in overall detection*
float32 confidence
